securesystemslib/metadata.py

```diff
@@ -97,3 +98,50 @@ def pae(self) -> bytes:
             len(self.payload),
             self.payload,
         )
+
+    def sign(self, signer: Signer) -> Signature:
+        """sign the payload and create the signature
+
+        Arguments:
+            signer: A "Signer" class instance
```

Comment:
```suggestion
        """Sign the payload and create the signature.

        Arguments:
            signer: A "Signer" class instance.
```
Please start the docstrings the way you write a sentence with an upper case symbol.